assets/src/scripts/components/hydrograph/parameters.js

```diff
@@ -121,33 +121,33 @@ export const addSparkLine = function(svgSelection, {seriesLineSegments, scales})
 
 
 /**
- * Draws a table with clickable rows of timeseries parameter codes. Selecting
+ * Draws a table with clickable rows of time series parameter codes. Selecting
  * a row changes the active parameter code.
  * @param  {Object} elem                        d3 selection
  * @param  {String} siteno
- * @param  {Object} availableTimeseries         Timeseries metadata to display
+ * @param  {Object} availableTimeSeries         Time series metadata to display
  * @param  {Object} lineSegmentsByParmCd        line segments for each parameter code
  * @param  {Object} timeSeriesScalesByParmCd    scales for each parameter code
  */
-export const plotSeriesSelectTable = function (elem, {siteno, availableTimeseries, lineSegmentsByParmCd, timeSeriesScalesByParmCd}) {
+export const plotSeriesSelectTable = function (elem, {siteno, availableTimeSeries, lineSegmentsByParmCd, timeSeriesScalesByParmCd}) {
     // Get the position of the scrolled window before removing it so it can be set to the same value.
-    const lastTable = elem.select('#select-timeseries table');
+    const lastTable = elem.select('#select-time-series table');
     const scrollTop = lastTable.size() ? lastTable.property('scrollTop') : null;
-    elem.select('#select-timeseries').remove();
+    elem.select('#select-time-series').remove();
 
-    if (!availableTimeseries.length) {
+    if (!availableTimeSeries.length) {
         return;
     }
 
     const columnHeaders = ['Parameter', 'Preview', '#'];
     const tableContainer = elem.append('div')
-        .attr('id', 'select-timeseries');
+        .attr('id', 'select-time-series');
     tableContainer.append('label')
-        .attr('id', 'select-timeseries-label')
+        .attr('id', 'select-time-series-label')
         .text('Select a timeseries');
```

Comment:
Missed this one

Reply:
Ha, probably one of the only ones in the frontend! 👍

Reply:
Yes, exactly! :)